spec/omnicache/store_spec.rb

```diff
@@ -63,79 +94,140 @@ def self.load(value)
 
     it "uses that serializer" do
       values = [1, 2, 3]
-      store.write("key", values)
+      store.write("key1", values)
+      store.write_multi({ "key2" => values })
       values << 4
-      expect(store.read("key")).to eq([1, 2, 3, 4])
```

Comment:
This test looks like it was originally wrong before your change. It seems like it should be testing that the returned array is the _exact same object_ as `values`